### DISCUSSION TITLE
Улучшения панели закладок

### DISCUSSION THREAD

**Discussion by ongots** [mbnuqw/sidebery#936] [General]
Панель закладок значительно повышает комфорт использования закладок, спасибо за эту функцию. Сейчас она раскрыта на половину возможностей.

Очень не хватает кнопки "Переместить в панель закладок" в контекстном меню вкладок.

Удалять открываемые закладки из "Панели закладок" по умолчанию или в виде опции.

`Конвертировать в панель закладок` > `Удалить неиспользуемые закладки` - Неочевидное действие. Почему закладки в панели "неиспользуемые" и зачем их удалять во время конвертации непонятно. Пару раз случайно удалял закладки, не понимая что означает это действие.

**Comment by mbnuqw** [comment]
Перемещение табов в закладки уже планируется - для целой панели, для выделенных табов, через контекстное меню, через перетаскивание (опционально).

`Удалять открываемые закладки из "Панели закладок" по умолчанию или в виде опции.`
Да, идея хорошая.

`Неочевидное действие`
Хорошо, подумаю как переименовать, может добавлю описание действий. Так же, этот попап планируется сделать опциональным (добавлю галку "запомнить и больше не спрашивать")

Переношу в discussions из-за общего характера этого тикета.
